client/app/views/folder.coffee

```diff
@@ -219,122 +219,60 @@ module.exports = class FolderView extends BaseView
         new ModalShareView
             model: @model
 
+
+
     ###
         Drag and Drop and Upload
     ###
     onDragStart: (event) ->
         event.preventDefault()
         event.stopPropagation()
 
+
     onDragEnter: (event) ->
         event.preventDefault()
         event.stopPropagation()
         if not @isPublic or @canUpload
             @uploadButton.addClass 'btn-cozy-contrast'
             @$('#files-drop-zone').show()
 
+
     onDragLeave: (event) ->
         event.preventDefault()
         event.stopPropagation()
         if not @isPublic or @canUpload
             @uploadButton.removeClass 'btn-cozy-contrast'
             @$('#files-drop-zone').hide()
 
+
     onDrop: (event) ->
         event.preventDefault()
         event.stopPropagation()
         return false if @isPublic and not @canUpload
 
-        # folder drag and drop is only supported in Chrome
```

Comment:
Folder drag and drop was compatible only with old versions of Chrome? It seems to be removed in these changes with the new Chrome versions, is it considered as a regression?